### PR TITLE
W-10742843: Update readme sample and refresh token example to use SLAS

### PR DESCRIPTION
**Pull request by gkhunger-sf**
* Update readme sample to use SLAS
* Update refresh token example to use SLAS

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -53,65 +53,83 @@ To use an SDK client, instantiate an object of that client and configure these p
...
 // Import the SDK in TypeScript
 // tsc requires the --esModuleInterop flag for this
-import * as CommerceSdk from "commerce-sdk";
+import { Search, Customer } from "commerce-sdk";
 // For Javascript, use:
 // import CommerceSdk from "commerce-sdk";
-const { ClientConfig, helpers, Search } = CommerceSdk;
+// const { Search, Customer } = CommerceSdk;
 // Older Node.js versions can instead use:
 // const { ClientConfig, helpers, Search } = require("commerce-sdk");
```
Comment (wjhsf) [inline README.md:66]: These comments are outdated. This is sufficient:
```ts
import { Search, Customer } from "commerce-sdk";
// Older Node.js versions can instead use:
// const { ClientConfig, helpers, Search } = require("commerce-sdk");
```

File: README.md [hunk 2]
```diff
@@ -53,65 +53,83 @@ To use an SDK client, instantiate an object of that client and configure these p
...
 
-// Create a configuration to use when creating API clients
+// demo client credentials, if you have access to your own please replace them below.
+// do not store client secret as plaintext. Store it in a secure location.
+const CLIENT_ID = "da422690-7800-41d1-8ee4-3ce983961078";
+const CLIENT_SECRET = "D*HHUrgO2%qADp2JTIUi";
+const ORG_ID = "f_ecom_zzte_053";
+const SHORT_CODE = "kv7kzm78";
+const SITE_ID = "RefArch";
+
```
Comment (wjhsf) [inline README.md:75]:
```suggestion
```
The README is a quick overview, it doesn't need a full working example.

File: README.md [hunk 3]
```diff
@@ -53,65 +53,83 @@ To use an SDK client, instantiate an object of that client and configure these p
...
-      organizationId: '<your-org-id>',
-      shortCode: '<your-short-code>',
-      siteId: '<your-site-id>'
-    }
+  headers: {},
+  parameters: {
+    clientId: CLIENT_ID,
+    organizationId: ORG_ID,
+    shortCode: SHORT_CODE,
+    siteId: SITE_ID,
```
Comment (wjhsf) [inline README.md:83]: If we don't include working credentials in the README, should we use `CLIENT_ID` or `"<your-client-id>"`?

File: README.md [hunk 4]
```diff
@@ -53,65 +53,83 @@ To use an SDK client, instantiate an object of that client and configure these p
...
+  },
+};
+
+/**
```
Comment (wjhsf) [inline README.md:87]: Can't suggest across the whole block, but here's a snippet just to tidy up and save a few lines.
```ts
const credentials = Buffer.from(`${CLIENT_ID}:${CLIENT_SECRET}`).toString("base64");
const loginClient = new Customer.ShopperLogin({
  // Headers can be specified in the client config and used for all requests
  headers: { authorization: `Basic ${credentials}` },
  ...config
});
// Get the shopper or guest JWT/access token, along with a refresh token, using client credentials
const token = await loginClient.getAccessToken({
  headers,
  body: { grant_type: "client_credentials" },
});

const searchClient = new Search.ShopperSearch(config);
// Search for dresses
const searchResults = await searchClient.productSearch({
  // Headers can also be provided per request
  headers: { authorization: `Bearer ${token.access_token}` },
  parameters: { q: "dress", limit: 5 }
});

if (searchResults.total) {
  const firstResult = searchResults.hits[0];
  console.log(`${firstResult.productId} ${firstResult.productName}`);
} else {
  console.log("No results for search.");
}
```

File: examples/03-refresh-auth-token.ts [hunk 5]
```diff
@@ -8,82 +8,82 @@
 /**
  * Get refresh token for a guest user
  * Usage: ts-node examples/03-refresh-auth-token.ts
- * Note: Replace configuration parameters before running
+ * For more information, see (Shopper Login and API Access Service)[https://developer.commercecloud.com/s/api-details/a003k00000VWfNDAA1/commerce-cloud-developer-centershopperloginandapiaccessservice].
```
Comment (wjhsf) [inline examples/03-refresh-auth-token.ts:11]:
```suggestion
 * For more information, see [Shopper Login and API Access Service](https://developer.commercecloud.com/s/api-details/a003k00000VWfNDAA1/commerce-cloud-developer-centershopperloginandapiaccessservice).
```
`[text](link)` is for markdown, but I don't know if this gets converted to markdown anywhere. 🤔

**Review by wjhsf — APPROVED** [review]

**Review by tarcang — APPROVED** [review]

**Review by wjhsf — APPROVED** [review]

**Pull request merged**